user/user.go

```diff
@@ -169,3 +169,14 @@ func SetUserIntoPolicyContractor(policy *models.Policy, origin string) {
 		lib.CheckError(err)
 	}
 }
+
+func GetAuthUserByMail(mail string) (models.User, error) {
+	var user models.User
+
+	authId, err := lib.GetAuthUserIdByEmail(mail)
```

Comment:
Even though both have the same value (as seen on consume-invite.go lines 60, 61), the function GetAuthUserIdByEmail returns the UID, not the authId.